quick-start/src/main/ui/app/entities/entity.model.ts

```diff
@@ -160,7 +160,7 @@ export class Entity {
         height: 100
       },
       'info': {
-        'baseUri': null,
+        'baseUri': 'http://example.org',
```

Comment:
http://example.org/, as we're doing this check:

readonly BASE_URI_REGEX = /^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]/;

Reply:
I changed it a little bit compared to what we have in 5x. I removed the last bit that expects trailing /
`+(\/)$`

Reply:
oh, for some reason I put that in. Whoops.